allennlp/service/server_sanic.py

```diff
@@ -9,22 +9,23 @@
 LOGGING['handlers']['accessTimedRotatingFile']['filename'] = '/tmp/sanic_access.log'
 LOGGING['handlers']['errorTimedRotatingFile']['filename'] = '/tmp/sanic_error.log'
 
-# TODO(joelgrus): make this configurable
-servables = ServableCollection.default()  # pylint: disable=invalid-name
 
 app = Sanic(__name__)  # pylint: disable=invalid-name
```

Comment:
Seems like `service` could be at the top directory level, rather than here, now that the cli is in `commands` anyway. +1.